keyboards/keysofkings/twokey/info.json

```diff
@@ -0,0 +1 @@
+{"keyboard": "keysofkings/twokey", "keymap": "default", "layout": "LAYOUT", "layers": [["LT(1, KC_MUTE)", "LT(4, KC_MPLY)", "LT(7, KC_MNXT)"], ["KC_TRNS", "TO(2)", "TO(3)"], ["TO(0)", "RGB_TOG", "RGB_MOD"], ["TO(0)", "RGB_HUI", "RGB_HUD"], ["TO(5)", "KC_TRNS", "TO(6)"], ["TO(0)", "RGB_SAI", "RGB_SAD"], ["TO(0)", "RGB_VAI", "RGB_VAD"], ["TO(8)", "TO(9)", "KC_TRNS"], ["TO(0)", "CK_TOGG", "MU_TOG"], ["TO(0)", "RESET", "EEPROM_RESET"]]}
```

Comment:
```suggestion
{
    "keyboard_name": "Twokey",
    "url": "",
    "maintainer": "Keys of Kings",
    "width": 2,
    "height": 2,
    "layouts": {
        "LAYOUT": {
            "layout": [
                {"x": 0, "y": 0},
                {"x": 0, "y": 1},
                {"x": 1, "y": 1}
            ]
        }
    }
}

```
![image](https://user-images.githubusercontent.com/4781841/97534916-e93ebf80-1a0e-11eb-9650-06836fcf804e.png)

Reply:
So the QMK Configurator will reference the keymap.c? 

This file was just generated from the c2json command. Looks like if removing everything it created is necessary then the command output needs a major rework.

Reply:
This file (info.json) is not a Configurator keymap export. It simply describes the layouts of a keyboard to the Configurator, so that it knows what order to place the keycodes in when generating the keymap.c, and how to display them in the UI.

The usual way to generate the info.json currently is to recreate each possible layout in KLE, then paste the contents of the Raw Data tab into https://qmk.fm/converter, and merge the outputs together (since each run will generate a complete info.json file).

See also: https://docs.qmk.fm/#/reference_info_json

Reply:
Thank you for the replies and links for more info. I edited the file, though I changed the X & Y coordinates you suggested to better represent the look of the (tiny) board.

Reply:
I forgot to mention you can also preview this file by hitting Ctrl+Shift+I in the Configurator. Turning on the Fast Input setting will also let you verify the ordering matches that of the LAYOUT macro.